### PR TITLE
mark files with invalid escape sequences as raw

### PR DESCRIPTION
**Pull request by gertingold**
This PR addresses the comment https://sourceforge.net/p/pyx/mailman/message/36732745/ on the pyx-user mailing list by marking strings with invalid escape sequences as raw. In one case, `\s` was replaced by `\\s` because the string contained also the valid escape sequences `\r` and `\w`.

### CODE REVIEW

File: pyx/mathutils.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def _realroots_quartic(a3, a2, a1, a0):
 
 
 def realpolyroots(*cs):
-    """returns the roots of a polynom with given coefficients
+    r"""returns the roots of a polynom with given coefficients
```
Comment (joerg-lehmann) [inline pyx/mathutils.py:112]: This seems to be an unnecessary raw string.

**Review by joerg-lehmann — COMMENTED** [review]
Many thanks, Gert! Looks good. Just one raw string (see comment) seems to be unnecessary. Maybe you can fix that before I merge the PR.

File: pyx/mathutils.py (hunk 1 above)
Comment (joerg-lehmann) [reply pyx/mathutils.py:112]: Sorry, my fault. All fine.!

Comment (gertingold) [reply pyx/mathutils.py:112]: Python 3.8 is complaining about the `\s` in `\sum` in the fourth line of the docstring. Therefore, it seems that we need to keep it.

**Comment by joerg-lehmann** [comment]
Sorry, my fault. All changes are fine. PR merged.

**Comment by gertingold** [comment]
You were faster than me...

**Pull request merged**